File: c++-problem-solving/baekjoon/leap-year-2753.cpp
#include <iostream>
#include "solutions.h"

using namespace std;

void leap_year_2753() {
    int n;
    cin >> n;

    bool answer = false;

    if(n%4 == 0) {
        if(n%100 != 0 || n % 400 == 0) {
            answer = true;
        }
    }

    cout << answer;
}

File: JavaProblemSolving/src/baekjoon/LeapYear2753.java
package baekjoon;

import java.util.Scanner;

public class LeapYear2753 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int year = scanner.nextInt();
        int answer = 0;

        if(year % 4 ==0){
            if(year % 100 != 0 || year % 400 == 0){
                answer = 1;
            }
        }

        System.out.println(answer);
    }
}
